src/components/annotation-form.jsx

```diff
@@ -17,79 +18,122 @@ import {Version} from '../services/web-monitoring-db';
-                    <Checkbox {...common} name="repeat_12">Scrolling news/reports</Checkbox>
-                </div>
-                <div className="col-md-4">
-                    <h4>Significance</h4>
-                    <Checkbox {...common} name="sig_1">Change related to energy, environment, or climate</Checkbox>
-                    <Checkbox {...common} name="sig_2">Language is significantly altered</Checkbox>
-                    <Checkbox {...common} name="sig_3">Content is removed</Checkbox>
-                    <Checkbox {...common} name="sig_4">Page is removed</Checkbox>
-                    <Checkbox {...common} name="sig_5">Insignificant</Checkbox>
-                    <Checkbox {...common} name="sig_6">Repeated Insignificant</Checkbox>
-                </div>
-            </form>
-        );
-    }
+        <Tooltip
+          id="annotation-tooltip"
+          place="top"
+          disable={!this.props.collapsed}
+          effect="solid" />
+      </form>
+    );
+  }
 
-    _onFieldChange (valueObject) {
-        if (this.props.onChange) {
-            const newAnnotation = Object.assign({}, this.props.annotation, valueObject);
-            this.props.onChange(newAnnotation);
-        }
+  _onFieldChange (valueObject) {
+    if (this.props.onChange) {
+      const newAnnotation = Object.assign({}, this.props.annotation, valueObject);
+      this.props.onChange(newAnnotation);
     }
+  }
+
+  _onNotesChange (event) {
+    this._onFieldChange({ notes: event.target.value });
+  }
 }
 
 AnnotationForm.defaultProps = {
-    annotation: null,
-    collapsed: true
+  annotation: null,
+  collapsed: true
 };
 
-function Checkbox ({children, formValues, name, onChange}) {
-    const fieldNumber = name.split('_')[1];
-    const checked = !!(formValues && formValues[name]);
-    const changeHandler = (event) =>
-        onChange({[name]: event.currentTarget.checked});
-
-    return (
-        <label>
-            <input type="checkbox" name={name} checked={checked} onChange={changeHandler} />
-            {fieldNumber}
-            <span className="info-text">= {children}</span>
-        </label>
-    );
+function Checkbox ({ children, formValues, name, onChange, collapsed }) {
+  const fieldNumber = name.split('_')[1];
+  const checked = !!(formValues && formValues[name]);
+  const uuid = getUUID();
+  const changeHandler = (event) =>
+    onChange({ [name]: event.currentTarget.checked });
+  return (
+    <li className="signifier-list-item">
+      <input
+        checked={checked}
+        className="toggle"
+        id={uuid}
+        onChange={changeHandler}
+        type="checkbox"
+        />
+      <label
+        className="toggle-btn"
+        data-for="annotation-tooltip"
+        data-tip={children}
+        data-fieldNumber={fieldNumber}
+        htmlFor={uuid}>
+          { collapsed ? "" : <span>{children}</span> }
```

Comment:
This is fine and there’s no need to change it back, but if you are working in more performance-sensitive areas in the future, it’s *dramatically* faster and more efficient to alter this with CSS than by adding or removing elements. (If you look at good scrolling list implementations or things like CodeMirror, for example, you’ll see they reuse elements that have moved off-screen and put new text in them.)